docs/shipping/Code/node-js.md

```diff
@@ -95,6 +145,35 @@ For serverless environments, such as AWS Lambda, Azure Functions, or Google Clou
   logger.sendAndClose();
   ```
 
+
+### Add opentelemetry context
+If you're sending traces with OpenTelemetry instrumentation (auto or manual), you can correlate your logs with the trace context. That way, your logs will have traces data in it, such as service name, span id and trace id (version >= `2.2.0`). 
+
+This feature is enabled by default, To disable it, set the `AddOtelContext` param in your handler configuration to `false`, like in this example:
+
+```javascript
+var logger = require('logzio-nodejs').createLogger({
+  token: 'token',
+  type: 'no-otel-context',
+  addOtelContext: false
+});
+```
+
+### Build and test locally
```

Comment:
I Don't think we need this in our official docs site, its supposed to be for github users